src/core/api/emit_seek_events.ts

```diff
@@ -49,18 +47,18 @@ export default function emitSeekEvents(
       return EMPTY;
     }
 
-    const isSeeking$ = observableFromEvent(mediaElement, "seeking")
-      .pipe(mapTo("seeking" as const));
-
-    const hasSeeked$ = observableFromEvent(mediaElement, "seeked").pipe(
+    const isSeeking$ = clock$.pipe(
+      filter((tick : IClockTick) => tick.state === "seeking" &&
+                                  !tick.isInternallySeeking),
```

Comment:
I think this condition is always true because in any case `tick.isInternallySeeking` is always reset to false when a `seeking' state is detected.

I see for example two possible solutions here:
  - `tick.isInternallySeeking` could still be equal to true when the clock's `isInternallySeeking` internal value had just be rest to false
  - the `state` property could be set to `internal-seek` instead, here we would have no need to use two properties outside the clock